src/features/form/components/AdminFormDetailPages/components/ExportDialog.module.css

```
@@ -0,0 +1,164 @@
+
+.exportPopupActions {
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	gap: 1rem;
+	flex-wrap: wrap;
+}
+
+/* 語意 class */
+.exportPrimaryButton {
+}
```

Stylelint (17.11.0)

[error] Empty block (block-no-empty)

(block-no-empty)